pennylane/qnode.py

```diff
@@ -671,6 +671,10 @@ def __call__(self, *args, **kwargs):
         ):
             return res
 
+        if self._qfunc_output.return_type is qml.measurements.Shadow:

```

Comment:
This will probably have to be revisited if the return types get merged before

Reply:
Indeed the first PR has already been merged, so I'll change this.

Reply:
It doesn't look like anything actually needs to be changed here,